homeassistant/components/vallox/__init__.py

```diff
@@ -0,0 +1,80 @@
+"""Support for Vallox ventilation units."""
+
+import ipaddress
+import logging
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_SENSORS
+from homeassistant.helpers import discovery
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'vallox'
+DEFAULT_NAME = 'Vallox'
+
+MODE_METRIC_KEY = 'A_CYC_MODE'
+
+DATA_VALLOX_SENSOR_LIST = 'vallox_sensor_list'
+
+# List of implemented sensors.
+SENSOR_FAN_SPEED = 'fan_speed'
+SENSOR_TEMP_EXTRACT_AIR = 'temp_extract_air'
+SENSOR_TEMP_EXHAUST_AIR = 'temp_exhaust_air'
+SENSOR_TEMP_OUTDOOR_AIR = 'temp_outdoor_air'
+SENSOR_TEMP_SUPPLY_AIR = 'temp_supply_air'
+SENSOR_HUMIDITY = 'humidity'
+SENSOR_REMAINING_TIME_FILTER = 'remaining_time_filter'
+
+SENSORS = [
+    SENSOR_FAN_SPEED,
+    SENSOR_TEMP_EXTRACT_AIR,
+    SENSOR_TEMP_EXHAUST_AIR,
+    SENSOR_TEMP_OUTDOOR_AIR,
+    SENSOR_TEMP_SUPPLY_AIR,
+    SENSOR_HUMIDITY,
+    SENSOR_REMAINING_TIME_FILTER
+]
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_HOST): vol.All(ipaddress.ip_address, cv.string),
+        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_SENSORS): vol.All(cv.ensure_list, [vol.In(SENSORS)]),
```

Comment:
We no longer want to include config options for selecting sensor types. All available types should be included by default.

See https://github.com/home-assistant/architecture/blob/master/adr/0003-monitor-condition-and-data-selectors.md.